salt/metalk8s/orchestrate/upgrade/init.sls

```diff
@@ -100,3 +100,9 @@ Deploy MetalK8s UI:
     - saltenv: {{ saltenv }}
     - require:
       - salt: Precheck for MetalK8s UI
+
+Make sure salt master container is up:
+  module.wait:
+    - cri.wait_container:
+      - name: salt-master
+      - state: running
```

Comment:
Honnestly it's not needed here salt-master should not restart during upgrade.
Also it's already done in `salt/metalk8s/salt/master/installed.sls` but maybe it's done too quickly after deploying the new pod so maybe we should add just a sleep before checking the status of the pod